lightning/src/ln/chan_utils.rs

```diff
@@ -334,8 +349,8 @@ pub struct ChannelPublicKeys {
 	/// counterparty to create a secret which the counterparty can reveal to revoke previous
 	/// states.
 	pub revocation_basepoint: PublicKey,
-	/// The public key which receives our immediately spendable primary channel balance in
-	/// remote-broadcasted commitment transactions. This key is static across every commitment
+	/// The public key which receives an immediately spendable primary channel balance in
+	/// a broadcaster's commitment transactions. This key is static across every commitment
```

Comment:
This reads a bit confusing - its the countersignatory's balance in a broadcaster's transaction, but reading it I'd think its a broadcaster's balance.